packages/SystemsThinkingWorkbench-Core.package/STWNode.class/instance/startDrag..st

```diff
@@ -0,0 +1,6 @@
+interaction
+startDrag: evt
+
+	self draggedConnection: (STWDragConnection new).
+	self owner addConnection: self draggedConnection.
```

Comment:
Why should we add this connection to the STWSimulation? We iterate over the connection collection while the simulation is running in order to perform the step logic for every connection. But a STWDragConnection have no logic so it is not necessary to add it to this collection.